test/e2e-upgrade-tests.sh

```diff
@@ -36,8 +36,48 @@ source $(dirname $0)/e2e-common.sh
 OPERATOR_DIR=$(dirname $0)/..
 KNATIVE_SERVING_DIR=${OPERATOR_DIR}/..
 
-function knative_setup() {
+function install_latest_operator_release() {
+  header "Installing Knative Serving operator latest public release"
+  local url="https://github.com/knative/serving-operator/releases/download/${LATEST_SERVING_RELEASE_VERSION}"
+  local yaml="serving-operator.yaml"
+
+  local RELEASE_YAML="$(mktemp)"
```

Comment:
should the var be lower case?
im assuming all local var using lower case for consistency

Reply:
Done.